scripts/resize.rb

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/ruby
+
+=begin
+A simple script for resizing images in bulk. Does a scaled resize.
+
+Maximum size is defined below with MAX_WIDTH and MAX_HEIGHT.
+
+Only processes .jpg files.
+
+Does not try to resize images below the defined `MAX_` dimensions.
+
+(I may have gone a bit overboard here, forgive me -IK)
```

Comment:
lol, yes, let's put:

Images do not need to be more than 1MB but we don't prevent larger ones from being uploaded.

A developer can do a bulk resize with [imagemagick](https://imagemagick.org/) from the root of this project `mogrify -resize 2000x2000 ./public/images/uploads/*.jpg`

This will replace all `.jpg` files with a version of the image that is max 2000 x 2000. It will not alter the aspect ratio. The majority will then be less than 1MB

in the readme rather than add more dependencies here.

Reply:
But will this upscale images smaller than that to 2000x2000 ?

Reply:
turns out

```
mogrify -resize '2000x2000>' ./public/images/uploads/*.jpg
```

does this